### PR TITLE
CRM 6 adding new migration and renaming previous one(incorrect number)

### PR DESCRIPTION
**Pull request by ell96p**

### CODE REVIEW

File: db-migration/src/main/resources/db/migration/V3__relational_table_customers_cars.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+create table customers_cars
+
```
Comment (Fellway) [inline db-migration/src/main/resources/db/migration/V3__relational_table_customers_cars.sql:2]: Please format this script.

File: db-migration/src/main/resources/db/migration/V3__relational_table_customers_cars.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+create table customers_cars
```
Comment (Fellway) [inline db-migration/src/main/resources/db/migration/V3__relational_table_customers_cars.sql:1]: CREATE TABLE IF NOT EXIST. Look at the previously created scripts

**Review by Fellway — CHANGES_REQUESTED** [review]

File: db-migration/src/main/resources/db/migration/V3__relational_table_customers_cars.sql (hunk 2 above)
Comment (ell96p) [reply db-migration/src/main/resources/db/migration/V3__relational_table_customers_cars.sql]: Ok

File: db-migration/src/main/resources/db/migration/V3__relational_table_customers_cars.sql (hunk 1 above)
Comment (ell96p) [reply db-migration/src/main/resources/db/migration/V3__relational_table_customers_cars.sql]: ok

**Review by Fellway — APPROVED** [review]

**Pull request merged**